### PR TITLE
Fix cardano-testnet help

### PR DESCRIPTION
**Pull request by Jimbo4350**
# Description

Prior to this fix the cardano-testnet help output was as follows:
```
cardano-testnet --help    
Usage: cardano-testnet (COMMAND | COMMAND)

Available options:
  -h,--help                Show this help text

Testnets:
  babbage                  
  byron                    
  cardano                  
  shelley                  

General:
  version          
```
We have improved it to:
```
cardano-testnet --help                                                                                             
Usage: cardano-testnet (cardano | byron | shelley | babbage | version)

Available options:
  -h,--help                Show this help text

Available commands:
  cardano                  Start a testnet in any era
  byron                    Start a Byron testnet
  shelley                  Start a Shelley testnet
  babbage                  Start a babbage testnet
  version                  Show cardano-testnet version
```

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated.  These may include:
  - golden tests
  - property tests
  - roundtrip tests
  - integration tests
  See [Runnings tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Any changes are noted in the `CHANGELOG.md` for affected package
- [ ] The version bounds in `.cabal` files are updated
- [ ] CI passes. See note on CI.  The following CI checks are required:
  - [ ] Code is linted with `hlint`.  See `.github/workflows/check-hlint.yml` to get the `hlint` version
  - [ ] Code is formatted with `stylish-haskell`.  See `.github/workflows/stylish-haskell.yml` to get the `stylish-haskell` version
  - [ ] Code builds on Linux, MacOS and Windows for `ghc-8.10.7` and `ghc-9.2.7`
- [ ] Self-reviewed the diff

# Note on CI
If your PR is from a fork, the necessary CI jobs won't trigger automatically for security reasons.
You will need to get someone with write privileges.  Please contact IOG node developers to do this
for you.

### CODE REVIEW

File: cardano-testnet/src/Testnet/Parsers.hs [hunk 1]
```diff
@@ -9,20 +10,28 @@ import           Parsers.Cardano
 import           Parsers.Shelley
 import           Parsers.Version
 
-{- HLINT ignore "Monoid law, left identity" -}
+-- TODO: Remove StartBabbageTestnet and StartShelleyTestnet
+-- by allowing the user to start testnets in any era (excluding Byron)
+-- via StartCardanoTestnet
+data CardanoTestnetCommands
+  = StartBabbageTestnet BabbageOptions
+  | StartByrontestnet ByronOptions -- TODO: Do we care about being able to start a Byron only testnet?
```
Comment (carloslodelar) [inline cardano-testnet/src/Testnet/Parsers.hs:18]: Probably not.

Comment (CarlosLopezDeLara) [inline cardano-testnet/src/Testnet/Parsers.hs:18]: Probably not, specially if it is too involved.

**Review by carbolymer — APPROVED** [review]

**Comment by CarlosLopezDeLara** [comment]
Wondering if this should be presented on a different order... i.e. 

```
Available commands:
  cardano                  Start a testnet in any era
  byron                    Start a Byron testnet
  shelley                  Start testnet in Shelley era
  babbage                  Start a babbage testnet
  version                  Show cardano-testnet version
```
or 

```
Available commands:
  byron                    Start a Byron testnet
  shelley                  Start testnet in Shelley era
  babbage                  Start a babbage testnet
  cardano                  Start a testnet in any era
  version                  Show cardano-testnet version
```

**Comment by Jimbo4350** [comment]
> Wondering if this should be presented on a different order... i.e.
> 
> ```
> Available commands:
>   cardano                  Start a testnet in any era
>   byron                    Start a Byron testnet
>   shelley                  Start testnet in Shelley era
>   babbage                  Start a babbage testnet
>   version                  Show cardano-testnet version
> ```
> 
> or
> 
> ```
> Available commands:
>   byron                    Start a Byron testnet
>   shelley                  Start testnet in Shelley era
>   babbage                  Start a babbage testnet
>   cardano                  Start a testnet in any era
>   version                  Show cardano-testnet version
> ```

Done :+1:

**Review by newhoggy — APPROVED** [review]

**Pull request merged**